Python/pylifecycle.c

```diff
@@ -511,6 +511,8 @@ pycore_create_interpreter(_PyRuntimeState *runtime,
     }
     *interp_p = interp;
 
+    _PyGC_Initialize(&interp->gc);
```

Comment:
Hum, maybe _PyGC_Initialize() should take interp argument instead, to be consistent with other _PyGC functions.